pandas/io/tests/test_excel.py

```diff
@@ -91,266 +86,173 @@ def read_csv(self, *args, **kwds):
         kwds['engine'] = 'python'
         return read_csv(*args, **kwds)
 
+    def get_data(self, basename, csv=True):
+        """
+        Return a DataFrame as read by the Python csv engine and a DataFrame
+        as read by the ExcelFile engine. Test data path is defined by
+        pandas.util.testing.get_data_path()
```

Comment:
Can this be more clear it returns 1 or 2 items?  Part of me thinks it should just be two functions.